docs/applications/sabnzbd.md

```diff
@@ -0,0 +1,11 @@
+# Sabnzbd
+
+Homepage: [https://sabnzbd.org/)
+
+The time tested Usenet downloader provided with FreeNAS. It just works for those migrating from FreeNAS.
+
+## Usage
+
+Set `sabnzbd_enabled: true` in your `inventories/<your_inventory>/nas.yml` file.
```

Comment:
Hello!

You do mention `sabnzbd_enabled: true` but I don't see it referenced in `all.yml`. It is true to say that it works without it (because of it's default value in `nas.yml`) but `all.yml` is used for reference of what is overridable in `inventories/<your_inventory>/nas.yml` and it is also used for Vagrant tests.

Reply:
Yep. My bad on that one.

Got it now.